library-vcpkg/CMakeLists.txt

```diff
@@ -18,15 +18,21 @@ endif()
 target_compile_definitions(
 	"${LIB_NAME}" PUBLIC
 	"DPP_BUILD"
-	"$<$<PLATFORM_ID:Windows>:$<$<CONFIG:Debug>:/sdl;/std:c++17;/Od;/DEBUG;/sdl;/MP;/DFD_SETSIZE=1024;/Zc:preprocessor>>"
+	"$<$<PLATFORM_ID:Windows>:$<$<CONFIG:Debug>:/sdl;/std:c++17;/Od;/DEBUG;/MP;/DFD_SETSIZE=1024;/Zc:preprocessor>>"
 	"$<$<PLATFORM_ID:Windows>:$<$<CONFIG:Release>:/std:c++17;/O2;/Oi;/Oy;/GL;/Gy;/sdl;/MP;/DFD_SETSIZE=1024;/Zc:preprocessor>>"
```

Comment:
Those two lines should be removed, shouldn't they? they are not compile definitions

Reply:
as part of this pr? no.

this pr is just to match the diff that is in the vcpkg repository, which was as simple a fix as possible. considering the hassle of making things build on vcpkg I wanted to keep it simple. further fixes can be additional PRs, as I mentioned in the pull request over there.